manifests/cf-manifest/spec/manifest/release_versions_spec.rb

```diff
@@ -42,6 +42,7 @@ def normalise_version(v)
 
     unpinned_cf_deployment_releases.each { |name, version|
       next if name.end_with? '-buildpack'
+      next if name == 'uaa'
```

Comment:
Like this:

```diff
diff --git a/manifests/cf-manifest/spec/manifest/release_versions_spec.rb b/manifests/cf-manifest/spec/manifest/release_versions_spec.rb
index 772c2c4b..94a5516b 100644
--- a/manifests/cf-manifest/spec/manifest/release_versions_spec.rb
+++ b/manifests/cf-manifest/spec/manifest/release_versions_spec.rb
@@ -26,7 +26,12 @@ RSpec.describe "release versions" do
       Gem::Version.new(v.gsub(/^v/, '').gsub(/^([0-9]+)$/, '0.0.\1'))
     end
 
-    pinned_releases = {}
+    pinned_releases = {
+      'uaa' => {
+        local: '0.0.1562849910',
+        upstream: '72.0',
+      }
+    }
 
     manifest_releases = manifest_without_vars_store.fetch("releases").map { |release|
       [release['name'], release['version']]
@@ -42,7 +47,6 @@ RSpec.describe "release versions" do
 
     unpinned_cf_deployment_releases.each { |name, version|
       next if name.end_with? '-buildpack'
-      next if name == 'uaa'
       next unless manifest_releases.has_key? name
       expect(normalise_version(manifest_releases[name])).to be >= normalise_version(version),
         "expected #{name} release version #{manifest_releases[name]} to be older than #{version} as defined in cf-deployment. Maybe you need to pin it?"
```

This will make the tests fail every time upstream's UAA release gets bumped, which should remind us to keep our fork up to date.